### PR TITLE
ASA - About This Data

### LINKED ISSUE #3854: ASA: Disclaimer - About this data
**Describe the task**
As a FBAN, I need to see where the data comes from and how thresholds are calculated so that I can understand how the Advisories are derived.

**Acceptance Criteria**
- [x] Data disclaimer is accessible by any user viewing ASA
- [x] A discrete rolloever tooltip is present at the top right of the screen 
- [x] Content to be easily interpreted by any user ( Intermediate/Conversational Reading Level)
- [x] Remove 'Percentage of combustible land threshold' slider

**Additional context**
- Inspired from the tooltip in the top right of the [HFI calculator](https://psu.nrs.gov.bc.ca/hfi-calculator)
- 20% threshold info for Advisories needs to be mentioned (Eric K to provide exact wording)
- [Figma mockup for MVP1 for this feature found here](https://www.figma.com/design/YbTS5mEgXgCApFkGnZqvfj/ASA-Desktop?node-id=904-233&t=KaVmeW3alFa8MIQi-4).

### PR DESCRIPTION
**Pull request by brettedw**
- Adds 'About This Data' button to ASA which currently just displays information about the advisory thresholds and a but about fuel types for future work
- Closes #3854 

# Test Links:
[Landing Page](https://wps-pr-3890-e1e498-dev.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-3890-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[Percentile Calculator](https://wps-pr-3890-e1e498-dev.apps.silver.devops.gov.bc.ca/percentile-calculator)
[C-Haines](https://wps-pr-3890-e1e498-dev.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3890-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3890-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3890-e1e498-dev.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3890-e1e498-dev.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

**Review by conbrad — APPROVED** [review]

File: web/src/features/fba/components/AboutDataPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          }}
+        >
+          About This Data
+        </Typography>
+        <div
+          style={{ paddingBottom: theme.spacing(3), paddingRight: theme.spacing(2), fontSize: '0.9rem' }}
+          data-testid="about-data-content"
+        >
+          <ul style={{ margin: 0 }}>
+            <li>
```
Comment (dgboss) [inline web/src/features/fba/components/AboutDataPopover.tsx:61]: The markdown for this list is broken in the deployment for some reason and I only see a single bullet point in the list.
image

Comment (dgboss) [reply web/src/features/fba/components/AboutDataPopover.tsx:61]: image

Comment (dgboss) [reply web/src/features/fba/components/AboutDataPopover.tsx:61]: Might need to make the entire string interpolated to get around it.

File: web/src/features/fba/components/AboutDataPopover.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        id="about-data-popover"
+        open={open}
+        anchorEl={anchorEl}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'left'
+        }}
+        onClose={handlePopoverClose}
+        disableRestoreFocus
+        slotProps={{ paper: { sx: { maxWidth: 350, backgroundColor: '#EEEEEE' } } }}
```
Comment (dgboss) [inline web/src/features/fba/components/AboutDataPopover.tsx:45]: Is this #EEEEEE color used anywhere else in the app? If so, perhaps we could import it? If not, I wonder if we should match a pre-existing color (eg. the background around the advisory bulletin box perhaps)?

**Review by dgboss — APPROVED** [review]
This looks good! Just a couple minor comments.

File: web/src/features/fba/components/AboutDataPopover.tsx (hunk 1 above)
Comment (brettedw) [reply web/src/features/fba/components/AboutDataPopover.tsx:61]: Huh, that's interesting. I see all the bullets locally

Comment (brettedw) [reply web/src/features/fba/components/AboutDataPopover.tsx:61]: Looks like just re-deploying took care of it....odd

Comment (dgboss) [reply web/src/features/fba/components/AboutDataPopover.tsx:61]: Very strange!

File: web/src/features/fba/components/AboutDataPopover.tsx (hunk 2 above)
Comment (dgboss) [reply web/src/features/fba/components/AboutDataPopover.tsx]: Thank you!

**Pull request merged**